Tests/HederaE2ETests/Schedule/ScheduleSign.swift

```diff
@@ -0,0 +1,162 @@
+/*
+ * ‌
+ * Hedera Swift SDK
+ * ​
+ * Copyright (C) 2022 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permiss  ions and
+ * limitations under the License.
+ * ‍
+ */
+
+import Hedera
+import XCTest
+
+internal final class ScheduleSign: XCTestCase {
+
+    internal func testTransferSign() async throws {
+        let testEnv = try TestEnvironment.nonFree
+
+        let key1 = PrivateKey.generateEd25519()
+        let key2 = PrivateKey.generateEd25519()
+
+        let key_list: KeyList = [.single(key1.publicKey), .single(key2.publicKey)]
+
+        // Create a new account with keylist
+        let accountReceipt = try await AccountCreateTransaction()
+            .key(.keyList(key_list))
+            .initialBalance(Hbar(1))
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let accountId = try XCTUnwrap(accountReceipt.accountId)
+
+        // Transfer, and sign with key1 upon creating schedule transaction
+        let receipt = try await TransferTransaction()
+            .hbarTransfer(accountId, Hbar(-1))
+            .hbarTransfer(testEnv.operator.accountId, Hbar(1))
+            .schedule()
+            .sign(key1)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let scheduleId = try XCTUnwrap(receipt.scheduleId)
+
+        // Sign schedules transaction
+        _ = try await ScheduleSignTransaction()
+            .scheduleId(scheduleId)
+            .sign(key2)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let info = try await ScheduleInfoQuery(scheduleId: scheduleId).execute(testEnv.client)
+
+        _ = try XCTUnwrap(info.executedAt)
+
+        // Check if signatures in schedule info
+        XCTAssertEqual(info.signatories, [.single(key1.publicKey), .single(key2.publicKey)])
+
+        _ = try await AccountDeleteTransaction()
+            .accountId(accountId)
+            .transferAccountId(testEnv.operator.accountId)
+            .sign(key1)
+            .sign(key2)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+    }
```

Comment:
Move this to when you first get the account ID, and put it in a `addTeardownBlock`